src/daidepp/__init__.py

```diff
@@ -1,7 +1,6 @@
-
-from daidepp.daide_visitor import daide_visitor
-from daidepp.grammar import create_daide_grammar
+from daidepp.grammar import *
```

Comment:
Wildcard imports are generally a no-no, though for something like a grammar file with a lot of objects I'd say an exception is well justified